packages/tree/src/Treemap.ts

```diff
@@ -1,6 +1,7 @@
 ﻿import { ITree } from "@hpcc-js/api";
-import { HTMLWidget, PropertyExt, Utility } from "@hpcc-js/common";
-import { hierarchy as d3Hierarchy, treemap as d3Treemap } from "d3-hierarchy";
+import { HTMLWidget, PropertyExt, textColor, Utility } from "@hpcc-js/common";
```

Comment:
textColor here looks wrong - import Palette instead and then use `Palette.textColor`